### PR TITLE
feat: Single Product page

### PR DESCRIPTION
**Pull request by Shivani-1524**
Added a single product page

### CODE REVIEW

File: SingleProduct/singleproduct.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                <button class="btn warning-outlined-btn icon-left mg-t-10">
+                    <i class="far fa-heart"></i>
+                    Add to WishList
+                </button>
+                <button class="btn primary-btn solid icon-left mg-t-10">
+                    <i class="fas fa-shopping-cart"></i>
+                    Add to Bag
+                </button>
+            </div>
+            <p class="sm-title mg-t-20">PRODUCT DETAILS</p>
```
Comment (gauravsinhaweb) [inline SingleProduct/singleproduct.html:86]: use text-transform uppercase instead of hard coding text.

**Review by gauravsinhaweb — APPROVED** [review]
UI looks so good. Good use of semantics as well.

File: SingleProduct/singleproduct.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            <a class="nav-link sm-title hide-md" href="../Products/products.html">Shop</a>
+            <a class="nav-link sm-title hide-md" href="../index.html">Categories</a>
+        </div>
+        <div class="nav-space"></div>
+        <div class="nav-icon-items">
+            <div class="nav-search-bar hide-sm">
+                <input class="search-bar" type="text" placeholder="Search..">
+                <i class="fa fa-brands fa-searchengin"></i>
+            </div>
+            <a href="../Authentication/login.html">
```
Comment (varun1233) [inline SingleProduct/singleproduct.html:30]: Suggestion: Instead of putting the button inside a tag you can put the i inside a tag

**Review by varun1233 — APPROVED** [review]
Great work. Code looks neat and good to merge. Just added one suggestion you can look into it.

**Review by bhtibrewal — APPROVED** [review]
Everything looks okay to me, good to merge

**Review by Naman-Saxena1 — APPROVED** [review]
Capture

If you want, you can make it responsive for small screens, everything else looks fine 👍

File: SingleProduct/singleproduct.html (hunk 1 above)
Comment (Shivani-1524) [reply SingleProduct/singleproduct.html:86]: okay!

**Comment by Shivani-1524** [comment]
> UI looks so good. Good use of semantics as well.

Thanks a lot

File: SingleProduct/singleproduct.html (hunk 2 above)
Comment (Shivani-1524) [reply SingleProduct/singleproduct.html:30]: okay will do!

**Comment by Shivani-1524** [comment]
> Great work. Code looks neat and good to merge. Just added one suggestion you can look into it.

Thanks Varun

**Pull request merged**